webapp/src/main/java/com/box/l10n/mojito/converter/LocalTimeConverter.java

```diff
@@ -16,6 +17,9 @@ public class LocalTimeConverter implements Converter<String, LocalTime> {
 
   @Override
   public LocalTime convert(String source) {
```

Comment:
is that converter still relevant now we're migrating the types? what is used for?